pages/spicedb/api/_meta.json

```
@@ -8,5 +8,10 @@
     "title": "HTTP API Reference",
     "href": "https://www.postman.com/authzed/workspace/spicedb/overview",
     "newWindow": true
+  },
+  "http-api": {
```

this is overriding the key from the HTTP reference
```suggestion
  "cloud-api": {
```